saltstack/final/salt/top.sls

```diff
@@ -11,6 +11,7 @@ final:
 {% if pillar['subtype'] != 'Docker' %}
     - cis-controls
 {% endif %}
+    - openscap
```

Comment:
I don't think that we need this on Docker

Reply:
yes it wast planned